File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let p_cnt = 0, y_cnt = 0;
    
    s.split('').forEach(e => {
        e === 'p' || e ==='P' ? p_cnt++ : null;
        e === 'y' || e === 'Y' ? y_cnt++ : null;
    })
    
    return p_cnt === y_cnt;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <bits/stdc++.h>
using namespace std;

bool solution(string s)
{
    int p_cnt = 0, y_cnt = 0;
  
    for(int i=0; i<s.length(); i++){
        if(s[i] == 'p' || s[i] == 'P') p_cnt++;
        else if(s[i] == 'y' || s[i] == 'Y') y_cnt++;
    }
    
    return p_cnt == y_cnt;

}